features/step_definitions/stripe_steps.rb

```diff
@@ -0,0 +1,17 @@
+Given("stripe is accepting payments") do
+end
+
+Then("a stripe customer was created for me") do
+  expect(app.current_user.stripe_customer_id).not_to be_nil
+  expect(app.stripe_api.created_customers.last).to include(tok: "fake-elements-token",
```

Comment:
I'm intending to wrap the Stripe Ruby Gem so that we can have a "fake client" that stores the the past transactions with it ala Hexagonal application architecture.